query/queryCompiler.js

```diff
@@ -103,6 +103,7 @@ function _getSelect() {
     if (_.has(this._grouped, "aggregate")) {
       _.each(this._grouped.aggregate, function (aggregate) {
         // TODO add.. more.. aggregates
+        var key, val;
```

Comment:
ESLint is unhappy about redefining previously defined `key`, `val` variables, hence I extracted definition to a common scope.